src/entities/area.py

```diff
@@ -35,14 +35,14 @@ def set_entity(self, properties):
             elif _type == StandardPropertyURN.BLOCKADES.name:
                 self.blockades.set_value(values[0].listInt.values)
 
-    # def get_apexes(self):
-    #     if self.apexes is None:
-    #         self.apexes = []
-    #         for edge in self.get_edges():
-    #             self.apexes.append(edge.get_start_x())
-    #             self.apexes.append(edge.get_start_y())
```

Comment:
@ffaraji was this commented for a reason or it's fine to un-comment it and use it?